docs/DeveloperGuide.md

```diff
@@ -97,6 +97,11 @@ The `UI` component,
 * depends on some classes in the `Model` component, as it displays `GameEntry` object residing in the `Model`. The graph
 and statistics displays also depend on `GameEntryList`.
 
+Below is a sequence diagram that shows how the UI parts interact 
+when a command is executed. (e.g. a delete command)
+
+![DeleteSequenceDiagram(UI)](images/DeleteSequenceDiagram(Ui).png)
```

Comment:
I can't seem to view the sequence diagram in the rendered file. Perhaps it could be a issue with the naming of the file? Should we consider naming it to DeleteSequenceDiagram_Ui.png or something along those lines?